x-pack/test/api_integration/deployment_agnostic/apis/observability/ai_assistant/utils/model_and_inference.ts

```diff
@@ -20,6 +20,7 @@ export const TINY_TEXT_EMBEDDING_MODEL_ID = SUPPORTED_TRAINED_MODELS.TINY_TEXT_E
 // tiny inference endpoints
 export const TINY_ELSER_INFERENCE_ID = 'pt_tiny_elser_inference_id';
 export const TINY_TEXT_EMBEDDING_INFERENCE_ID = 'pt_tiny_text_embedding_inference_id';
+export const ELSER_INFERENCE_ID = '.elser-2-elasticsearch';
```

Comment:
You can import this from https://github.com/elastic/kibana/blob/9f6eb0a0cb7408a614335773240e3b360c9eceee/x-pack/platform/plugins/shared/observability_ai_assistant/common/preconfigured_inference_ids.ts#L10